core/metadata/src/model/ModuleDescriptor.ts

```
@@ -0,0 +1,27 @@
+export class ModuleDescriptorDumper {
+  static dumpPath(desc: ModuleDescriptor) {
+    const dumpDir = DUMP_PATH || desc.unitPath;
+    return path.join(dumpDir, '.egg', `${desc.name}_module_desc.json`);
+  }
+
+  static dump(desc: ModuleDescriptor) {
+    const dumpPath = ModuleDescriptorDumper.dumpPath(desc);
+    fs.mkdirSync(path.dirname(dumpPath), { recursive: true });
+    fs.writeFileSync(dumpPath, JSON.stringify(desc));
+  }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)